notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #84.            dataflow_machine_type=dataflow_machine_type,
</pre><p>Ditto here from other NB. Do we want to shows users these fields in this notebook? From lines 84 to 89. They are more advanced functionality &amp; I don't see explanations around these fields. Maybe a callout to the docs page about additional dataflow configuration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
Removed the dataflow parameters from the code. There's 
![image](https://user-images.githubusercontent.com/45586340/192496944-d073e385-dcda-4f9d-8f2f-b029442a469a.png)
this link which has explanation of each of these components.